infra/docker-images/db-ops/anonymize.sql

```diff
@@ -1,5 +1,3 @@
-CREATE EXTENSION pgcrypto;
```

Comment:
`CREATE EXTENSION if not exists pgcrypto;`
au cas où ?

Reply:
Pour l'instant, ça n'est pas nécessaire. Le dump de la prod exporte ce `CREATE EXTENSION`, donc l'extension aura été installée lors du restore.

Si jamais on supprime l'extension en prod, effectivement, ce script cassera.

D'un autre côté, si on laisse ce `CREATE EXTENSION` ici, et qu'on supprime l'extension en prod, alors on aura une différence entre staging et prod. (Et si jamais on a supprimé l'extension par erreur, le code fonctionnera sous staging car l'extension sera là, mais pas sur la prod où l'extension n'y sera plus.)

Le top serait d'installer l'extension dans ce script et de la désinstaller seulement si elle n'était pas déjà installé auparavant (par le restore, et donc en prod). C'est peut-être possible, mais je ne sais pas faire présentement et je suis moyennement motivé. :)